config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
@jrafanie IIRC, it failed to store my data if it was over 1M and I didn't set `value_max_bytes`.